src/bp/ui-shared/src/FormFields/TextFieldsArray/index.tsx

```diff
@@ -57,6 +58,18 @@ const TextFieldsArray: FC<TextFieldsArrayProps> = props => {
     onChange([...newItems])
   }
 
+  const validateItem = (item: string) => {
+    if (!validationPattern?.test) {
+      return null
+    }
+
+    return validationPattern.test(item) ? (
+      <Icon icon="tick-circle" className={cx(style.icon, style.success)}></Icon>
```

Comment:
We should not render any icon if example is empty (I see in your gif that there's a tick sign when input is empty)

Reply:
Good point